apps/aws-notebook/manifest.yaml

```diff
@@ -0,0 +1,9 @@
+daemons:
+  openvpn:
+    run:
+      cmd: |
+        set -ex
+        sudo service dbus start
+        openvpn3  session-start --config $OPENVPN_CONFIG_PATH
+        # Then session will run in background
+        sleep infinity
```

Comment:
Is it possible to keep openvpn3 in the foreground? My practice is directly using openvpn, like:

```sh
openvpn --config "$OPENVPN_CONFIG_FILE"
```

It's good enough for everything, even don't need dbus.